### DISCUSSION TITLE
Super slow page loading and API responses today

### DISCUSSION THREAD

**Discussion by brandonscript** [imRohan/Pantry#168] [Q&A] [answered]
Hey there, getting really slow responses today both in site performance and APIs. Wondering if something might be up?

**Comment by imRohan** [comment]
You're right, performance in the last few weeks has been a bit slow. I've been looking into it and it seems like Pantry now has enough hourly traffic to warrant a wide-spread rate-limiting layer. 

There are many users who are constantly performing [GET] requests and flooding the server. I'll have to look into preventing this somehow.

**Comment by brandonscript** [comment]
Well, hey that’s a good problem to have! Rate-limiting should be pretty easy to get up and running. Good luck to you!

**Comment by imRohan** [comment, marked as answer]
Deployed a fix this morning, Pantry should feel a bit snappier now. ⚡
